### PR TITLE
Automations docs

### PR DESCRIPTION
**Pull request by cicdw**

### CODE REVIEW

File: docs/orchestration/concepts/automations.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
+# Automations <Badge text="Cloud"/>
+
+Automations allow you to configure actions (such as cancelling a flow run or sending a notification to certain endpoints) when your flows or agents encounter certain events.
+
+For example, you can send a Slack message to your team when a run from a production critical flow has failed, along with the reason for the failure so that you can respond immediately. Or you can cancel a run that has been running (or scheduled) for more than an hour to enforce SLAs.  The Prefect Cloud API currently supports notification actions for Slack, Twilio, Pager Duty, email and arbitrary webhooks.
+
+You can set up and edit your Automations using the [API](/orchestration/concepts/api.html) or the Automations page in the UI.
+
+## Events
```
Comment (zangell44) [inline docs/orchestration/concepts/automations.md:9]: what is an `Event`?

File: docs/orchestration/concepts/automations.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+Currently you can configure automations for the following events:
+- A flow run enters a given state or states
+- Runs from multiple (or all!) flows enter a given state or states
+
+Additionally, if you are a Standard Tier user, you can configure automations for the following scenarios:
+- A flow run fails to start after being scheduled for a certain amount of time
+- A flow run fails to finish after running for a certain amount of time
+- All agents with the same agent-config-id become unhealthy
+
+## Actions
```
Comment (zangell44) [inline docs/orchestration/concepts/automations.md:20]: what is an `Action`?

Comment (madkinsz) [reply docs/orchestration/concepts/automations.md:20]: Note this also covers a small subset of available actions and we should probably add the rest

Comment (mashun4ek) [reply docs/orchestration/concepts/automations.md:20]: Yes, we should expand this:
Action is a response on the [event](/orchestration/concepts/automations.md#events). You can send notifications to following services:
- Slack
- Twilio
- PagerDuty
- MS Teams
- Email
- Custom webhook
- Prefect UI
Moreover, you can 
- Cancel flow run

Comment (madkinsz) [reply docs/orchestration/concepts/automations.md:20]: The PrefectMessage / UI notification is not available publically rn.

File: docs/orchestration/concepts/automations.md (hunk 1 above)
Comment (cicdw) [reply docs/orchestration/concepts/automations.md:9]: Updated w/ a sentence

**Review by zangell44 — DISMISSED** [review]

File: docs/orchestration/concepts/automations.md (hunk 2 above)
Comment (cicdw) [reply docs/orchestration/concepts/automations.md]: @madkinsz how would you prefer I document these additional hooks slash would you like to take over this branch?

Comment (madkinsz) [reply docs/orchestration/concepts/automations.md]: I can do it tomorrow

Comment (madkinsz) [reply docs/orchestration/concepts/automations.md]: @zhen0 -- I'm pretty swamped with Jim out. Could you take this back over?

Comment (madkinsz) [reply docs/orchestration/concepts/automations.md]: We could also add a note that documentation for other actions is coming soon and just merge this as is.

Comment (zhen0) [reply docs/orchestration/concepts/automations.md]: Thanks for the updates. I'll take a look at this tomorrow.

**Comment by madkinsz** [comment]
@zhen0 let me know when this is ready -- planning to release today or tomorrow

**Review by madkinsz — APPROVED** [review]

**Pull request merged**